spyre/server.py

```diff
@@ -7,7 +7,7 @@
 import numpy as np
 
 import model
-import View
+import view
```

Comment:
i addressed this earlier today with these commits:
https://github.com/adamhajari/spyre/commit/f9ea089754b87355a957b6b7b46e7fc48884e016
https://github.com/adamhajari/spyre/commit/4bc535f06a938ac4efd5f61332440ca5c0fb5231

There's an issue with the packaging process where the "View.py" version of the filename is being cached and can't be changed to "view.py".  Changing all instances of View -> view will result in an error for anyone who installs using pip. 

It should suffice to change the filename from view.py -> View.py (which i did in the first commit above), but I also wrapped the import in a try/catch block just in case. I thought I did this weeks ago, apologies to everyone.